src/tests/Microsoft.PowerFx.Core.Tests.Shared/FormatterTests.cs

```diff
@@ -256,6 +260,8 @@ public void TestMinificationWithCommentRemovalInTextFirst(string script, string
         [InlineData("Set(str, $\"{{}}\")", "Set(\n    str,\n    $\"{{}}\"\n)")]
         [InlineData("Set(additionText, $\"The sum of 1 and 3 is {{{1 + 3}}})\")", "Set(\n    additionText,\n    $\"The sum of 1 and 3 is {{{1 + 3}}})\"\n)")]
         [InlineData("$\"This is {{\"Another\"}} interpolated {{string}}\"", "$\"This is {{\"Another\"}} interpolated {{string}}\"")]
+        [InlineData("ParseJSON(\"[]\", Type([{Age: Number}]))", "ParseJSON(\n    \"[]\",\n    Type([{Age: Number}])\n)")]

```

Comment:
>Type([{Age: Number}]) [](http://example.com/codeflow?start=94&length=21)

Should the type declaration be pretty-printed as well? Something like `Type([{ Name: Text, Age: Number, Address: { Street: Text, City: Text, State: Text, Zip: Text }, PhoneNumber: [{ Value: Number }]}])` can get really long, so it may be good to break it up

Reply:
it pretty prints if its long (more than one field) it was getting too long . thats why i added the next test